templates/Web/Pages/Angular.List/src/app/app-shell/Param_SourceName_Kebab/list.component.html

```diff
@@ -4,9 +4,7 @@
       <h3>Angular AngularList Template</h3>
     </div>
     <div class="col-12 p-0">
-      <app-list-form
-        (inputText)="handleAddListItem($event)"
-      ></app-list-form>
+      <app-list-form (inputText)="handleAddListItem($event)"></app-list-form>
```

Comment:
Is it an intentional change to move it to one line or it is something done automatically by prettier?

Reply:
automatically by prettier